scss/civicrm/contact/pages/_export-contact.scss

```diff
@@ -36,10 +53,15 @@
     }
   }
 
-  .crm-submit-buttons {
-    > .crm-button {
-      margin-left: 0 !important;
-      margin-right: 10px !important;
+  .crm-form-block {
+    background-color: transparent;
+    box-shadow: none;
+
+    > .crm-submit-buttons {
+      box-shadow: 0 8px 18px 0px rgba(0, 0, 0, .1);
```

Comment:
It seems that we are using this shadow on multiple places can this be converted to variable?